docs/sources/panels-visualizations/visualizations/table/index.md

```diff
@@ -68,16 +68,34 @@ The following video provides a visual walkthrough of the options you can set in
 
 {{< docs/play title="Table Visualizations in Grafana" url="https://play.grafana.org/d/OhR1ID6Mk/" >}}
 
+### Key Features:
+- **Multi-Mode Support**: Seamlessly visualize time series data, detailed tables, annotations, or raw JSON inputs.
+- **Advanced Formatting**: Customize your data presentation with options for date formatting and value formatting.
+- **Coloring Options**: Enhance readability and insights by applying custom coloring schemes to your data points.
+- **Diverse Cell Types**: Elevate your data display with a variety of cell types, including: Colored text and backgrounds, Gauges, JSON, Images, Sparklines, and multiple combinations.
+
 ## Annotation and alert support
 
-Annotations and alerts are not currently supported in tables.
+{{% admonition type="note" %}}
+Annotations and alerts are not currently supported for tables.
+{{% /admonition %}}
```

Comment:
A heading must be followed by some body text. I'm okay with making this a note, but it should either be a note without a heading or plain text with a heading. Do you think people need this information enough to keep a heading? If not, then we should remove the heading and leave it as a note. Also just updating the format of the admonition.
```suggestion
{{< admonition type="note" >}}
Annotations and alerts are not currently supported for tables.
{{< /admonition >}}
```